llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -761,8 +761,10 @@ Value *VPInstruction::generate(VPTransformState &State) {
     return Builder.CreatePtrAdd(Ptr, Addend, Name, getGEPNoWrapFlags());
   }
   case VPInstruction::AnyOf: {
-    Value *A = State.get(getOperand(0));
-    return Builder.CreateOrReduce(A);
+    Value *Res = State.get(getOperand(0));
+    for (VPValue *Op : drop_begin(operands()))
+      Res = Builder.CreateOr(Res, State.get(Op));
+    return Builder.CreateOrReduce(Res);
   }
```

Comment:
Worth documenting the distinction between AnyOf and ComputeAnyOfResult?

Reply:
I added some documentation for ComputeAnyOfResult in 772eb075674b413fe1409cf10c6dc58d560004c5, using the wording from IVDescriptors.h, thanks